lib/matplotlib/tri/_tripcolor.py

```diff
@@ -54,8 +55,25 @@ def tripcolor(ax, *args, alpha=1.0, norm=None, cmap=None, vmin=None,
         values used for each triangle are from the mean c of the triangle's
         three points. If *shading* is 'gouraud' then color values must be
         defined at points.
-    other_parameters
-        All other parameters are the same as for `~.Axes.pcolor`.
+    %(cmap_doc)s
+
+    %(norm_doc)s
+
+    %(vmin_vmax_doc)s
+
+    %(colorizer_doc)s
+
```

Comment:
What do you mean by standard?

Reply:
"Other Parameters" comes after "Returns" in the numpydoc sections order (https://numpydoc.readthedocs.io/en/latest/format.html#other-parameters). We do not have a strict policy on what goes where, but kwargs are typically in "Other Parameters", which makes sense, because they are rarely used and the list of artist properties is long.